src/pages/ChemicalDetails.vue

```diff
@@ -35,15 +41,27 @@ export default {
 
   },
   computed: {
+    
+    chemData: function() {
+      var lookup = this.chemName;
+      var items = this.$store.getters.chemicals;
+      var result = [];
+      for(var item, i=0; item=items[i++];) {
+        if ((item.chemical_name).toLowerCase() == lookup.toLowerCase()) {
+          result.push(item);
+        }
+      }
+      return result;
+    },
     columns: function() {
-      if (this.$store.getters.chemicals.length == 0)
+      if (this.chemData.length == 0)
```

Comment:
You could clean this up and make it one line if you use a logical OR apparently:
https://stackoverflow.com/questions/476436/is-there-a-null-coalescing-operator-in-javascript
so it'd be

//when chemData[0] == null left hand side evaluates to false, so JS uses the right hand side.
return Object.Keys(this.chemData[0]) || []

Reply:
done